src/main/java/org/fundacionjala/virtualassistant/models/RequestEntity.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.virtualassistant.models;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "request")
+public class RequestEntity {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long idRequest;
```

Comment:
this should be a Long wrapper value, whenever you define an entity in the persistence layer all the fields should have non primitive data types